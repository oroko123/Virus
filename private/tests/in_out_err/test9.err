Error in line 2:inny znaczek 018 1999 Poczta Polska
Error in line 3:jeszcze inny znaczek 199.99 001999 Poczta Polska
Error in line 4:ladny znaczek 007 00001999 Poczta Polska
