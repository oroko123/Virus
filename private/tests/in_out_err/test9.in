fajny znaczek 2000 1999 Poczta Polska
inny znaczek 018 1999 Poczta Polska
jeszcze inny znaczek 199.99 001999 Poczta Polska
ladny znaczek 007 00001999 Poczta Polska
1990 2000
