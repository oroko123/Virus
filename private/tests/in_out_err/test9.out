1999 Poczta Polska 2000 fajny znaczek
